### PR TITLE
fix stylus paths sort order to prioritize direct dependencies over de…

### PR DESCRIPTION
**Pull request by lambertsteven**
…pendencies of dependencies.

### CODE REVIEW

**Comment by lambertsteven** [comment]
@spacerockzero @redbugz

**Comment by lambertsteven** [comment]
Validated that it works with http://fs-styleguide-beta.herokuapp.com/ (the `background: red` of `html` is only on my node_modules/fs-styles and not part of theme-engage/node_modules/fs-styles)

File: lib/assets.js [hunk 1]
```diff
@@ -445,6 +445,12 @@ sproto.preprocessContent = function() {
     var stylusPaths = searchPaths.map(function(aPath) {
       return path.join(aPath, 'css');
     });
+
+    // stylus reads file paths bottom to top, so we need to reverse the order of our paths
+    // to get the repos direct dependencies at the bottom and then dependencies of dependencies
+    // closer to the top
+    utils.breathFirstPathSort(stylusPaths, 'desc');
```
Comment (spacerockzero) [inline lib/assets.js]: "breadth"

**Comment by lambertsteven** [comment]
fixed the typo.

**Pull request merged**